os/arch/arm/src/imxrt/imxrt_flexram.h

```diff
@@ -0,0 +1,436 @@
+/****************************************************************************
+ *
+ * Copyright 2019 NXP Semiconductors All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/*
+ * Copyright 2017 NXP
+ * All rights reserved.
+ *
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef _IMXRT_FLEXRAM_H_
+#define _IMXRT_FLEXRAM_H_
+
+#include <stdint.h>
+#include <tinyara/config.h>
+
+#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#include <chip/imxrt102x_features.h>
+#include "chip/imxrt102x_config.h"
+#elif defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT105x)
+#include <chip/imxrt105x_features.h>
+#include "chip/imxrt105x_config.h"
+#else
+#error Unrecognized i.MX RT architecture
+#endif
+
+#include "imxrt_config.h"
+
+/*!
+ * @addtogroup flexram
+ * @{
+ */
+
+/******************************************************************************
+ * Definitions.
+ *****************************************************************************/
+/*! @brief flexram write read sel */
+enum _flexram_wr_rd_sel
+{

```

Comment:
coding rules.

Reply:
Let's apply it later with another commit.